radwave/src/Radwave.vue

```diff
@@ -883,11 +925,31 @@ export default defineComponent({
           id: layer.id.toString(),
           settings: [
             ["color", Color.load(this.sunColor)],
-            ["scaleFactor", 100]
+            ["scaleFactor", 100],
           ]
         });
         return layer;
       });
+
+      const sunSquareLayerProm = this.createTableLayer({
+        referenceFrame: "Sky",
+        name: "Radcliffe Wave Sun Square",
+        dataCsv: text
+      }).then(layer => {
+        this.basicLayerSetup(layer);
+        this.applyTableLayerSettings({
+          id: layer.id.toString(),
+          settings: [
+            ["color", Color.load(this.sunColor)],
+            ["scaleFactor", 12000000000],
+            ["plotType", PlotTypes.square],
+            ["markerScale", MarkerScales.world]
```

Comment:
```suggestion
            ["scaleFactor", 30],
            ["plotType", PlotTypes.square],
            ["markerScale", MarkerScales.screen]
```
marker should scale with screensize.

Reply:
I did set this to world scale on purpose - I wasn't sure how weird it would seem to have the square marker size not depends on the zoom. But if we want it to always be the same size, I'm fine with that.